docs/products/tools/marketplace/guides/linuxgsm/index.md

```diff
@@ -0,0 +1,65 @@
+---
+description: "Deploy LinuxGSM on a Linode Compute Instance. LinuxGSM is a command line utility for managing multiplayer game servers."
+keywords: ['game servers','multiplayer','game']
+tags: ["marketplace", "linode platform", "cloud manager"]
+published: 2024-01-12
+modified_by:
+  name: Linode
+title: "Deploy LinuxGSM through the Linode Marketplace"
+external_resources:
+- '[LinuxGSM](https://linuxgsm.com/)'
+authors: ["Linode"]
+---
+
+[LinuxGSM](https://linuxgsm.com/) 
+
+## Deploying a Marketplace App
+
+{{< content "deploy-marketplace-apps-shortguide">}}
+
+{{< content "marketplace-verify-standard-shortguide">}}
+
+{{< note >}}
+**Estimated deployment time:** LinuxGSM should be fully installed within 10-15 minutes after the Compute Instance has finished provisioning.
+{{< /note >}}
+
+## Configuration Options
+
+- **Supported distributions:** Ubuntu 22.04 LTS
+- **Recommended plan:** All plan types and sizes can be used.
```

Comment:
Replace "Recommended" with "Suggested"

Reply:
Every single marketplace app guide uses the phrase "recommended plan." I can create another PR to change if necessary.

Reply:
We discussed this and determined it is not necessary to retroactively change the language "recommended plan" in existing guides.